mllib-tests/v1p4/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -160,6 +166,18 @@ class GLMRegressionTest(sc: SparkContext) extends GLMTests(sc) {
         val rr = new RidgeRegressionWithSGD().setIntercept(addIntercept = true)
         rr.optimizer.setNumIterations(numIterations).setStepSize(stepSize).setRegParam(regParam)
         rr.run(rdd)
+      case ("l2", "elastic-net") =>
+        println("WARNING: Liner Regression with elastic-net in ML package ignores numIterations, " +
+          "stepSize in this Spark version.")
```

Comment:
I modified it in the code. We may want to refactor it to use some static version string instead of hard coding the version.